templates/openwisp2/settings.py

```diff
@@ -368,3 +384,17 @@
 {{ instruction }}
 
 {% endfor %}
+
+{% if openwisp2_monitoring %}
+TIMESERIES_DATABASE = {
+    'BACKEND': '{{ openwisp2_timeseries_database.backend }}',
+    'USER': '{{ openwisp2_timeseries_database.user }}',
+    'PASSWORD': '{{ openwisp2_timeseries_database.password }}',
+    'NAME': '{{ openwisp2_timeseries_database.name }}',
+    'HOST': '{{ influxdb_http_ip }}',
+    'PORT': '{{ influxdb_http_port }}',
+}
+
+INSTALLED_APPS.append('djcelery_email')
+EMAIL_BACKEND = 'djcelery_email.backends.CeleryEmailBackend'
```

Comment:
We should have a separate setting for managing `djcelery_email`. The setting should be `true` by default.

Reply:
we can add a variable like `openwisp2_email_backend` which defaults to `djcelery_email.backends.CeleryEmailBackend`

Reply:
Done (https://github.com/openwisp/ansible-openwisp2/pull/262/commits/59796a1e231670bffeb084e5a890fda4fc64a3b8)